README.md

```diff
@@ -1,4 +1,106 @@
 # hacktoberbot
-A community project for Hacktoberfest 2018. A Discord bot primarily designed to help teach Python learners from the PythonDiscord community how to contribute to open source.
+A community project for [Hacktoberfest 2018](https://hacktoberfest.digitalocean.com). A Discord bot primarily designed to help teach Python learners from the PythonDiscord community how to contribute to open source.
 
 You can find our community by going to https://discord.gg/python
+
+## Motivations
+We know it can be difficult to get into the whole open source thing at first. To help out, we've decided to start a little community project during hacktober that you can all choose to contribute to if you're finding the event a little overwhelming, or if you're new to this whole thing and just want someone to hold your hand at first.
+
+*Lemon*
+
+## Commands
```

Comment:
I think we should simplify this list. perhaps instead of using h3s and listing aliases, we should just have a list with codeblocked commands (including prefix) in them. We should also make sure to use the imperative mood (just like a good docstring) when we describe them.

* `!repository` - Link to the hacktoberbot GitHub repo.
* `!git` - Link to the Getting Started with Git page.
* `!git.commit` - Demonstrate how to do a git commit with an example.

Reply:
would be great if these had the backticks before and after, to make the commands stand out better.

`!repository`, not !repository